### PR TITLE
Add Github Actions to lint and publish

### PR DESCRIPTION
**Pull request by krisgesling**
#### Description
This adds the new Mycroft standard Github Actions.
1. For all PRs lint with Flake8
2. For merged PRs to tag a release on git, and publish to PyPI.

Currently there are no unittests included in MSK so the standard installation and run of pytest has been removed. I've left the python version matrix in but with only a single version. This can be easily extended as tests are added.

In order to pass the Flake8 tests, the repo had to be thoroughly cleaned up. 

#### Type of PR
- [x] Feature implementation
- [x] Refactor of code (without functional changes)

#### Testing
As no unit tests exist, this should be tested manually. To assist with this I've set the version to be a release candidate.

#### CLA
- [x] Yes

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
```
Comment (forslund) [inline .github/workflows/pr.yml]: I think we should add at least 3.7 as well

**Review by forslund — APPROVED** [review]
Looks good. My only comment is that we should test on at least 3.7 as well since that is what
's used on the mark-1.

Also is double quotes (") a requirement now? This repo seemed to use single-qoutes (') quite consistently and afaik they are just as valid according to the standard?

**Comment by krisgesling** [comment]
No definitely not a requirement, autopep8 just wasn't handling lots of the code, so I did a pass with black, then autopep8 again. Even then there was a bunch of stuff that required manual fixing but a LOT less. 

So if anyone is wedded to single quotes we can do this update more carefully - but assuming there's no strong opinions either way, I'm not keen to do it again :D

I'll add 3.7 though

_Edit: since they run in parallel anyway, I've just re-added them all_

**Comment by krisgesling** [comment]
This also highlighted that we didn't have the master branch protections on for this repo. They've now been added.

**Comment by forslund** [comment]
Ok, probably just me preferring single-qoutes since it makes context switching from C/C++ easier. It's still approved so feel free to merge when you're ready.

**Pull request merged**